File: solutions/26_Remove_Duplicates_From_Sorted_Array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if len(nums) <= 1:
            return len(nums)

        nums_distinct = 0
        for num in nums:
            if num != nums[nums_distinct]:
                nums_distinct += 1
                nums[nums_distinct] = num
        return nums_distinct+1

File: solutions/26_Remove_Duplicates_From_Sorted_Array.c
int removeDuplicates(int* nums, int numsSize) {
    if (numsSize == 0) {
        return 0;
    } else if (numsSize == 1) {
        return 1;
    }

    int len = 1;
    for (int i = 1; i < numsSize; i++) {
        if (nums[i] != nums[len - 1]) {
            nums[len] = nums[i];
            len++;
        }
    }
    return len;
}